__tests__/__theia__/extension.theia.test.ts

```diff
@@ -9,51 +9,266 @@
 *                                                                                 *
 */
 
-import { Builder, By, Key, until } from "selenium-webdriver";
-// tslint:disable-next-line:no-submodule-imports
-import * as firefox from "selenium-webdriver/firefox";
 import * as chai from "chai";
 import * as chaiAsPromised from "chai-as-promised";
+import * as driverFF from "../../src/theia/extension.theiaFF";
```

Comment:
Kinda prefer having the word Firefox in the variable name, as when I was reading the test I didn't know what FF was.

Reply:
I have mentioned Firefox now everywhere